src/main.cpp

```diff
@@ -7795,38 +7787,6 @@ bool static ProcessMessage(const CChainParams& chainparams, CNode* pfrom, string
     }
 
 
-    else if (fAlerts && strCommand == "alert")
```

Comment:
This will now fall through to line 7892 where it is intentionally ignored without logging. That is the same behaviour as for the "notfound" message, so it does not introduce any new behaviour to the P2P protocol that an adversary could not already trigger.